tools/docker-dev/sql/bootstrap.sql

```diff
@@ -120,6 +120,16 @@ CREATE TABLE `groupRoles` (
   `perms` varchar(1000) NOT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
 
+--
+-- Dumping data for table `groupRoles`
+--
+
+INSERT INTO `groupRoles` (`id`, `name`, `slug`, `priority`, `color`, `perms`) VALUES
```

Comment:
For the `groupRequests` and the `groupAttributes` table, the start time and end time's default value should be _null_ instead of 0000, which defaults to the current time.